modules/mapbox/src/deck-utils.ts

```diff
@@ -300,7 +300,12 @@ type MaplibreRenderParameters = {
 
 function getViewport(deck: Deck, map: Map, renderParameters?: unknown): Viewport {
   const viewState = getViewState(map);
-  const view = getDefaultView(map);
+  const {
+    props: {views}
+  } = deck;
```

Comment:
Nit: I personally like to keep an eye on the auto formatter to avoid it breaking into too many lines.
```suggestion
  const {views} = deck.props;
```